packages/collector/src/announceCycle/unannounced.js

```diff
@@ -126,6 +127,7 @@ function applyAgentConfiguration(agentResponse) {
   applyExtraHttpHeaderConfiguration(agentResponse);
   applyKafkaTracingConfiguration(agentResponse);
   applySpanBatchingConfiguration(agentResponse);
+  applyIgnoreEndpointsConfiguration(agentResponse);
```

Comment:
QS: Did you manually test the flow with a real agent and check if the logic works?

Reply:
yes

Reply:
Sweet thank u